addons/common/backwards_comp.sqf

```diff
@@ -71,3 +71,11 @@ CBA_fnc_intToString = {
     if (isNil "_int") exitWith {""};
     str _int
 };
+
+CBA_fnc_inArea = {
```

Comment:
This won't be compileFinal:ed, which, if I'm not mistaken, CfgFunctions functions are.